pattern-1/dockerfiles/mysql/build.sh

```diff
@@ -18,19 +18,21 @@
 #
 # ------------------------------------------------------------------------
 
+set -e
+
 # Product Context
 product=wso2ei
 productVersion=6.1.1
 patternNumber=1
 # MySQL parameters
-rdbms=mysql
+database=mysql
```

Comment:
Shouldn't this be termed as rdbms since what we are building here is a relational database management system container runtime ?

Reply:
Yes, I think rdbms makes more sense! Will revert back. Thanks!